src/components/ViewRoutingSlip/TransactionDataTable.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <div>
+    <div class="header-bg-color d-flex align-center py-5 mb-0">
+      <v-icon color="primary" class="ml-8">
+          mdi-view-list
+      </v-icon>
+      <p class="ml-2 mb-0 font-weight-bold" data-test="title">Transactions</p><p class="mb-0" v-if="invoiceCount">{{ `(${invoiceCount})` }}</p>
+    </div>
+    <v-data-table
+    :headers="headerTranscations"
+    :items="invoiceDisplay"
+    class="elevation-1"
+    hide-default-footer
+    disable-pagination
+    >
+      <template v-slot:[`header.createdOn`]="{ header }">
+      <div class="pl-4">
+        {{ header.text }}
+      </div>
+      </template>
+      <template v-slot:[`item.createdOn`]="{ item }">
+      <div class="font-weight-bold pl-4">
+        {{  formatDisplayDate(item.createdOn, 'MMM DD, YYYY') }}
+      </div>
+    </template>
+    <template v-slot:[`item.total`]="{ item }">
+      <div class="font-weight-bold">
+        {{ `$${item.total}` }}
```

Comment:
i will do another dollar filter for this... for showing $$

Reply:
Added a common util method that accepts currency and appends $